Misc/NEWS.d/next/Build/2018-03-07-00-34-14.bpo-33015.AcryR_.rst

```diff
@@ -0,0 +1,3 @@
+Add a wrapper for thread function in PyThread_start_new_thread
+
+PyThread_start_new_thread accepts a function of type `void (*) (void *)`, which does not match with the pthread_create function callback prototype `void () (void *)`. This results in an invalid function cast warning with gcc8. Fix this by wrapping the function in an internal pthread function callback that returns NULL.
```

Comment:
I think that NEWS entries are usually short and don't explain the details (those are in the commit message), so I'd rephrase it (see other NEWS entries for examples).